run.py

```diff
@@ -428,7 +431,23 @@ def modifyonedrive(type):
         _winreg.CloseKey(onedrivekey)
         print "OneDrive key succesfully modified."
     except (WindowsError, IOError):
-        print "Unable to modify OneDrive key. Deleted, or is the program not elevated?"
+        print "Unable to modify OneDrive FileSync key. Deleted, or is the program not elevated?"
+        
+    try:
+        odxkey32 = _winreg.OpenKey(_winreg.HKEY_LOCAL_MACHINE, odxkey32, 0, _winreg.KEY_ALL_ACCESS)
```

Comment:
It's supposed to be here, replacing _winreg.HKEY_LOCAL_MACHINE with _winreg.HKEY_CLASSES_ROOT